src/components/view/ListView.vue

```diff
@@ -68,7 +68,7 @@
         <os-logo v-if="record.ostypename" :osName="record.ostypename" size="1x" style="margin-right: 5px" />
         <console :resource="record" size="small" style="margin-right: 5px" />
 
-        <span v-if="$route.path.startsWith('/globalsetting')">{{ text }}</span>
+        <span v-if="$route.path.startsWith('/globalsetting')"></span>
         <span v-if="$route.path.startsWith('/alert')">
```

Comment:
@ravening would be better to make this a v-else-if and leave the prior line as is since we don't need a link and detail view for each global setting

Reply:
@davidjumani can explain more? I didnt understand about the v-else-if part

Reply:
can change it to
```
         <span v-if="$route.path.startsWith('/globalsetting')">{{ text }}</span>
         <span v-else-if="$route.path.startsWith('/alert')">
```
So that the name is plain text and not a router link to a detail view of the global setting which is pretty much just the name

Reply:
@davidjumani the line `<span v-if="$route.path.startsWith('/globalsetting')">{{ text }}</span>` is displaying both plain text and the router link as shown in the screenshot above

Reply:
@davidjumani done